apps/showcase/scripts/update-screenshots/readme.md

```diff
@@ -0,0 +1,14 @@
+# How to generate screenshots for the visual testing
+
+To minimize discrepancies on the screenshots between different operating systems, we are generating the screenshots from inside a docker.
+The procedure is the following:
+
+1. Prerequisite: you need a docker setup (or podman), `docker` should be available globally
+2. Build your application with your latest modifications `yarn nx build showcase`.
+3. Serve the application on port 8080 `npx http-server ./apps/showcase/dist/browser`\
+     The script will look on all local interfaces for an app running on `http://localhost:8080`
```

Comment:
you might want to add an exception in `.github/markdown-external-links.config.json` for localhost.

Reply:
done

Reply:
This tool is considering what is between ``` ` ``` as link? Oo